lib/jekyll-plugin-platoniq-journal/documents_tag.rb

```diff
@@ -0,0 +1,81 @@
+# frozen_string_literal: true
+
+module Jekyll
+  class DocumentsTag < Liquid::Tag
+    include JekyllPluginPlatoniqJournal::Base
+    include JekyllPluginPlatoniqJournal::IncludesFile
+
+    def initialize(tag_name, input, tokens)
+      super
+      @input = input
+    end
+
+    def render(context)
+      @context = context
+      @site = site
+
+      @site.inclusions[include_file_path] ||= locate_include_file(include_file_path)
+
+      add_include_to_dependency(inclusion, context) if site.config["incremental"]
+
+      @icon = nil
+      context.stack do
+        @icon = inclusion.render(context)
+      end
+
+      render_all
+    end
+
+    private
+
+    def jdata
+      @jdata ||= JSON.parse(@input) if !@input.nil? && !@input.empty?
+    end
+
+    def include_file_path
+      "svg/icon-document.liquid"
+    end
+
+    def render_all
+      output = []
+
+      output << %(<section class="pj-links">)
```

Comment:
Should the css class be `pj-documents`?

```suggestion
      output << %(<section class="pj-documents">)
```